app/src/androidTest/java/com/github/h3lp3rs/h3lp/H3lpAppTest.kt

```diff
@@ -0,0 +1,144 @@
+package com.github.h3lp3rs.h3lp
+
```

Comment:
Though I love the name H3lpAppTest, I think that TestUtils or something in that vein would be more explicit

Reply:
Considering this is a superclass from which the test inherit, I don't agree. Although the class contains utilitary functions, the test classes _are_ H3lpAppTests, the name you suggest would fit if we _import_ it to use utilitary functions.